api/src/main/java/javax/config/spi/Converter.java

```diff
@@ -62,6 +62,20 @@
  * All Built In Converters have a {@code javax.annotation.Priority} of 1
  * A Converter should handle null values returning either null or a valid Object of the specified type.
  *
+ * <h3>Implicit Converters</h3>
+ *
+ * <p>If no explicit Converter and no built-in Converter could be found for a certain type,
+ * the {@code Config} provides an <em>Implicit Converter</em>, if</p>
+ * <ul>
```

Comment:
I'm not sure it matters in practice but should we specify the order of the implicit converters?
(i.e. check for `valueOf` before `parse`).

Your list looks fine so replacing the `<ul>` by an `<ol>` should be enough for that :)

Reply:
I gave this more thought. I still think we should follow the JAX-RS queryparam to just cover valueOf(String) and Constructor(String). Otherwise, as @jmesnil say, we have to specify a lot of orders to search the methods through. How useful is it?